core/string/string_name.h

```diff
@@ -136,6 +136,11 @@ class StringName {
 		// this is why path comparisons are very fast
 		return _data == p_name._data;
 	}
+	_FORCE_INLINE_ bool operator!=(const StringName &p_name) const {
+		// the real magic of all this mess happens here.
+		// this is why path comparisons are very fast
```

Comment:
The comments aren't from me :)

Reply:
@Ivorforce I see. But it would be good to fix the comments as they have been touched (moved).
At least I've seen this practice in other PRs :-)

Reply:
Then probably the above comment should also be changed, since it is an exact copy of this one?

Reply:
> Then probably the above comment should also be changed, since it is an exact copy of this one?

Not sure.
Lets call @AThousandShips who definitely knows how it should be done :)

Reply:
It's appreciated if moved or copied code is fixed, generally it's good to fix comments in code already touched by a commit

Reply:
Ok, fine by me. I modified both comments to match :)